modules/nix-darwin/default.nix

```diff
@@ -0,0 +1,92 @@
+# This method of generating Nix configuration borrows heavily from the nix-darwin project:
+# https://github.com/nix-darwin/nix-darwin/blob/0d71cbf88d63e938b37b85b3bf8b238bcf7b39b9/modules/nix/default.nix
```

Comment:
You will want to split this into two files: one with the nix-darwin code, one with our stuff, because nix-darwin is MIT (and this repo is unlicensed, probably a mistake, but to be addressed later) and the MIT license says:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
Good call, ideally even a directory with the MIT bits directly, with a copy of the upstream LICENSE file next to it.